roles/update-host/README.md

```diff
@@ -0,0 +1,47 @@
+# update-hosts role
+
+This role drive when the role updates the packages on the inventory
+
+## Role Variables
+
+`pkg_update: Boolean` defaults: False
```

Comment:
Can we make this a table - looks better that way. 3 columns: Name, What, Default Value